examples/pre_put_check/main.go

```diff
@@ -55,16 +55,24 @@ func main() {
 		}
 
 		fmt.Println("Item size:", size, "bytes")
+		// I wonder what the performance cost of this Is
+		// I think thsi really make sens to add to hologram if the performance cost is low
+		//  could do cool things as catching early or dynamically split objects.
 		if size > sizeLimit {
 			fmt.Printf("Item size is too large. Skipping item ID %v.", item.ID)
 			continue
-		} else {
-			fmt.Printf("Item size is within limits. Writing item ID %v.", item.ID)
 		}
+		// we dont need the else statement
+		fmt.Printf("Item size is within limits. Writing item ID %v.", item.ID)
 
-		PutItem(context.TODO(), client, &dynamodb.PutItemInput{
+		// What is the goal of this put? I think the example would be fine wih a fmt.Print here or similar
+		// Should we check the error?
+		if _, err := PutItem(context.TODO(), client, &dynamodb.PutItemInput{
```

Comment:
The goal is just to have a close-to-working example for anyone who would like to use this. As comment says above, you can just drop-in your actual client instead of the mock one, and it would work.

Should definitely check the error though :)